example/z-docker/.dockerignore

```diff
@@ -1,4 +1,39 @@
+# ocamlbuild working directory
 _build/
 _esy/
+
+# ocamlbuild targets
+*.byte
```

Comment:
Please also minimize this `.dockerignore`.

Reply:
Done